scripts/linters/pylint_extensions.py

```diff
@@ -2113,7 +2284,13 @@ def _process_module_tokens(self, tokens, node):
                 'redundant-type-comment', line=comment_line_number, node=node)
 
 
-class SingleSpaceAfterKeyWordChecker(checkers.BaseChecker):
+# TODO(#15613): Here we use MyPy ignore because of the incomplete typing of
```

Comment:
I'm a bit confused why #15613 issue used here, I think #15613 is filed for Apache beam stub. I would suggest to create a new issue for this pylint library.

Reply:
Yeah my bad. I have created a new issue #16567